apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx

```diff
@@ -37,6 +49,13 @@ export function TransferNFTForm({ objectId }: { objectId: string }) {
     const queryClient = useQueryClient();
     const navigate = useNavigate();
     const { clientIdentifier, notificationModal } = useQredoTransaction();
+
+    const kioskContents = useGetOriginByteKioskContents(activeAddress);
+    const transferKioskItem = useTransferKioskItem({ objectId, objectType });
+    const isContainedInKiosk = kioskContents?.data?.some(
```

Comment:
Is there no better way to tell this? Like GetObject on the owner and look at the type or something? With pagination this current approach won't be guaranteed to work.